pkg/app/app.go

```diff
@@ -65,11 +68,13 @@ func New() *App {
 	app.Flag("web.max-connection-age", "If set this limits the maximum lifetime of persistent HTTP connections.").
 		Default("0s").DurationVar(&cfg.maxConnectionAge)
 
+	reg := prometheus.NewRegistry()
```

Comment:
Creating a new registry, prevents showing the standard golang metrics (of memory, cpu, goroutines) and also doesn't show the metrics of the pulsar library which uses the DefaultRegisterer. 

Ideally we could register the standard metrics (`NewGoCollector` and `NewProcessCollector`) and pass the registry to pulsar (I haven't checked if pulsar supports it)

Reply:
I'll look into this.